test/games/strategy/triplea/ui/TestRouteOptimizer.java

```diff
@@ -0,0 +1,87 @@
+package games.strategy.triplea.ui;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.awt.Point;
+import java.awt.geom.Point2D;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import games.strategy.triplea.ui.mapdata.MapData;
+
+public class TestRouteOptimizer {
+
+  private MapPanel mapPanel;
+  private MapData mapData;
+
+  @Before
+  public void setUp() {
+    mapPanel = mock(MapPanel.class);
```

Comment:
This can be replaced with a @Mock on the variable declaration. NBD either way. I think most of our other mockito mocks are created that way though.